src/cpp/src/visual_language/qwen2vl/classes.cpp

```diff
@@ -292,6 +301,125 @@ ov::Tensor merge_text_and_image_embeddings(
     
 } // namespace qwen2vl_utils
 
+std::shared_ptr<ov::Model> VisionEncoderQwen2VL::patch_preprocess_into_model(std::shared_ptr<ov::Model> model_org) {
```

Comment:
The function doesn't use anything from `VisionEncoderQwen2VL1`. Move the function to anonymous namespace

Reply:
Sure, Thanks!

Reply:
I meant into `namespace {}`. That way the compiler knows that no one calls the function outside of this translation unit.

Reply:
OK, changed!